src/mesh/config/MeshConfiguration.cpp

```diff
@@ -113,7 +113,7 @@ void MeshConfiguration::addMesh(
         break;
       }
     }
-    PRECICE_ASSERT(found, "Data " << dataNewMesh->getName() << " is not defined. Please define a data tag with name=\"" << dataNewMesh->getName() << "\".");
+    PRECICE_CHECK(found, "Data {0} is not defined. Please define a data tag with name=\"{0}\".", dataNewMesh->getName());
```

Comment:
Why do you use `{0}` here and not `{}` like above?

Reply:
Like in python, I use `{0}` to repeatedly refer to the first argument.
Otherwise I would have to pass the data name twice.

Reply:
Ah ok. Makes sense :)